fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt

```diff
@@ -141,39 +145,47 @@ internal class HttpClient(
             headers.remove(Headers.CONTENT_LENGTH)
             contentLength = -1
         }
-
+        val inputStream = stethoHook?.interpretResponseStream(connection.inputStream)
+                ?: connection.inputStream
+        stethoHook?.postConnect()
```

Comment:
I prefer `hooks.postConnect`.

Are you sure this is where this should be called? Headers are already retrieved. Maybe it's meant as post received `Connection` which happens to have the headers, then it is fine. I just don't know about it 😁